modular_skyrat/modules/families/code/crayon_procs.dm

```diff
@@ -0,0 +1,59 @@
+GLOBAL_LIST(gang_tags)
+
+/obj/item/toy/crayon/proc/can_claim_for_gang(mob/user, atom/target, datum/antagonist/gang/user_gang)
+	var/area/A = get_area(target)
+	if(!A || (!is_station_level(A.z)))
+		to_chat(user, span_warning("[A] is unsuitable for tagging."))
+		return FALSE
+
+	var/spraying_over = FALSE
+	for(var/obj/effect/decal/cleanable/crayon/gang/G in target)
+		spraying_over = TRUE
+
+	for(var/obj/machinery/power/apc in target)
+		to_chat(user, span_warning("You can't tag an APC."))
+		return FALSE
+
+	var/obj/effect/decal/cleanable/crayon/gang/occupying_gang = territory_claimed(A, user)
+	if(occupying_gang && !spraying_over)
+		if(occupying_gang.my_gang == user_gang.my_gang)
+			to_chat(user, span_danger("[A] has already been tagged by our gang!"))
+		else
+			to_chat(user, span_danger("[A] has already been tagged by a gang! You must find and spray over the old tag instead!"))
+		return FALSE
+
+	// stolen from oldgang lmao
+	return TRUE
+
+/obj/item/toy/crayon/proc/tag_for_gang(mob/user, atom/target, datum/antagonist/gang/user_gang)
+	for(var/obj/effect/decal/cleanable/crayon/old_marking in target)
+		qdel(old_marking)
```

Comment:
add a break?

Reply:
i put a break here but it's not flagging as outdated